### ISSUE TITLE
capabilities.json missing in alpha package (3.0a5)

### ISSUE THREAD

**Issue by om26er** [python-escpos/python-escpos#343]
* install latest alpha build of the library
* try to import it
```
>>> from escpos import printer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/dist-packages/escpos/printer.py", line 21, in <module>
    from .escpos import Escpos
  File "/usr/local/lib/python3.7/dist-packages/escpos/escpos.py", line 48, in <module>
    from .magicencode import MagicEncode
  File "/usr/local/lib/python3.7/dist-packages/escpos/magicencode.py", line 23, in <module>
    from .codepages import CodePages
  File "/usr/local/lib/python3.7/dist-packages/escpos/codepages.py", line 1, in <module>
    from .capabilities import CAPABILITIES
  File "/usr/local/lib/python3.7/dist-packages/escpos/capabilities.py", line 40, in <module>
    with open(capabilities_path) as cp, open(pickle_path, 'wb') as pp:
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.7/dist-packages/escpos/capabilities.json'
```

**Comment by patkan** [comment]
You are right, I can reproduce this. :-/
As a quickfix you can add the path with the [environment variable `ESCPOS_CAPABILITIES_FILE`](https://python-escpos.readthedocs.io/en/latest/user/usage.html#advanced-usage-change-capabilities-profile).

As a real fix, I will have to think about how to package this more reliable. The way it is done at the moment it also makes trouble with CI-jobs...

**Comment by patkan** [comment]
I assume that #345 should fix the problem. I will read a bit more and then build a package and release it so you can check.

**Comment by patkan** [comment]
This should be fixed in v3.0a6
